config/default-spinnaker-local.yml

```diff
@@ -103,5 +116,8 @@ services:
         account: # twilio account id
         token: # twilio auth token
         from: # phone number by which sms messages are sent
+      slack:
+        enabled: false
+        token: # the part after http://hooks.slack.com/services/
 
```

Comment:
Is this an independent change?
slack isnt tied to CF, is it?
If it is the other way around, the slack change can get in there first anyway.